synapse/models/inet.py

```diff
@@ -940,12 +940,27 @@ def getBaseModels():
                     ('email', {'ptype': 'inet:email'}),
                     ('phone', {'ptype': 'tel:phone'}),
                     ('signup', {'ptype': 'time', 'doc': 'The time the web account was registered'}),
-                    ('signup:ipv4', {'ptype': 'inet:ipv4', 'doc': 'The original ipv4 address used to sign up for the account'}),
+                    ('signup:ipv4', {'ptype': 'inet:ipv4',
+                                     'doc': 'The original ipv4 address used to sign up for the account'}),
                     ('passwd', {'ptype': 'inet:passwd', 'doc': 'The current passwd for the web account'}),
                     ('seen:min', {'ptype': 'time:min'}),
                     ('seen:max', {'ptype': 'time:max'}),
                 ]),
 
+                ('inet:web:chprofile', {}, [
+                    ('acct', {'ptype': 'inet:web:acct', 'doc': 'Account associated with the change', 'ro': 1,
+                              'req': 1}),
+                    ('acct:site', {'ptype': 'inet:fqdn', 'doc': 'Site the user account is at', 'ro': 1, }),
+                    ('acct:user', {'ptype': 'inet:user', 'doc': 'User account name', 'ro': 1, }),
+                    ('ipv4', {'ptype': 'inet:ipv4', 'doc': 'Source IPv4 Address the account was done from', }),
```

Comment:
'Source IPv4 address used to make the account change.'
'Source IPv6 address used to make the account change.'

Reply:
fixed